docs/testing.md

```diff
@@ -139,37 +139,53 @@ The following steps are necessary to have a proper environment setup, where all
     Kubernetes cluster.
 
 
-3. Export the `CF_OPERATOR_WEBHOOK_SERVICE_HOST` env variable
+1. Export the `CF_OPERATOR_WEBHOOK_SERVICE_HOST` env variable
 
     ```bash
-    export CF_OPERATOR_WEBHOOK_SERVICE_HOST=$(minikube ssh -- "cat /etc/resolv.conf | grep nameserver | awk '{ printf \$2 }'")
+    export CF_OPERATOR_WEBHOOK_SERVICE_HOST=$(minikube ip)
     ```
 
     _**Note**_: You can also find the correct IP, by running `ip addr`. The IP address under `vboxnet1` is the IP that you need.
 
-4. Export the `OPERATOR_TEST_STORAGE_CLASS` env variable
+1. Export the `OPERATOR_TEST_STORAGE_CLASS` env variable
 
     ```bash
     export OPERATOR_TEST_STORAGE_CLASS=standard
     ```
 
     _**Note**_: Require for the PVC test creation, in minikube.
 
-5. Ensure `GO111MODULE` is set
+1. Copy the ssh-key required for tunneling to /tmp
+
+    ```bash
+    cp $(minikube ssh-key) /tmp/cf-operator-tunnel-identity
+    ```
+
+1. Allow port tunneling on minikube
+    ```bash
+    minikube ssh -- "sudo sed -i 's/#GatewayPorts no/GatewayPorts yes/' /etc/ssh/sshd_config && sudo systemctl reload sshd"
+    ```
+
+1. Enable tunneling from the minikube node to the cf-operator on localhost (required for webhooks)
+    ```bash
+    export ssh_server_user=docker
+    ```
+
+1. Ensure `GO111MODULE` is set
```

Comment:
Is that still needed? I use go 1.13 and don't set this variable. The `vendor` folder is still very helpful for docker builds, even with `GOPROXY` set.

Reply:
I have `GO111MODULE=on` by default, so I didn't notice whether it is still required or not. I can remove it, if you prefer.

Reply:
It defaults to `auto` and detect the `go.mod` and just work.

Anyhow, it's not really something you changed in the docs, so we can do this later.